WordPress/Classes/Services/AccountService.m

```diff
@@ -335,13 +330,6 @@ - (void)setupAppExtensionsWithDefaultAccount
                                      andOAuth2Token:oauth2Token];
         });
     }
-    
-    // Share Extension Configuration
-    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Derp, that was a total mental error on my part. I read that as `configureTodayExtension` so I removed it because it wasn't doing anything related to the today widget. Reverting the deletion of this. Great catch, thanks!